Content.Server/Cluwne/CluwneSystem.cs

```diff
@@ -0,0 +1,120 @@
+using Content.Server.Administration.Commands;
+using Content.Server.Speech.Components;
+using Content.Server.Popups;
+using Content.Shared.Popups;
+using Content.Server.Interaction.Components;
+using Content.Server.Speech;
+using Content.Shared.Mobs;
+using Robust.Shared.Random;
+using Robust.Shared.Audio;
+using Content.Shared.IdentityManagement;
+using Content.Shared.Stunnable;
+using YamlDotNet.Serialization;
+
+namespace Content.Server.Cluwne;
+
+public sealed class CluwneSystem : EntitySystem
+{
+    [Dependency] private readonly PopupSystem _popupSystem = default!;
+    [Dependency] private readonly SharedAudioSystem _audio = default!;
+    [Dependency] private readonly IRobustRandom _robustRandom = default!;
+    [Dependency] private readonly SharedStunSystem _stunSystem = default!;
+
+
+    public override void Initialize()
+    {
+        base.Initialize();
+
+        SubscribeLocalEvent<CluwneComponent, MobStateChangedEvent>(OnMobState);
+        SubscribeLocalEvent<CluwneComponent, ComponentStartup>(OnComponentStartup);
+    }
+
+    private void OnMobState(EntityUid uid, CluwneComponent component, MobStateChangedEvent args)
+    {
+        // removes cluwne component if cluwne dies.
+        if (args.NewMobState == MobState.Dead)
```

Comment:
Also ensure that they take a LOT of cellular damage when they die, something like
```cs
                var damageSpec = new DamageSpecifier(PrototypeManager.Index<DamageTypePrototype>("Genetic"), 300);
                DamageableSystem.TryChangeDamage(uid, damageSpec);
```
should work (Tho you should test if something stupid like 9999 instead of 300 damage works)

Reply:
Cluwne now suffers 300 cellular damage on death meaning they have 0% chance to be cloned.